src/main/java/Duke/DukeRunner.java

```diff
@@ -0,0 +1,63 @@
+
+// import java.util.Arrays;
+import java.util.Scanner;
+
+import Handlers.TaskManager;
+import Handlers.Parser;
+import Handlers.StorageManager;
+public class DukeRunner {
+
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?\n");
+        System.out.println("==============================\n");
```

Comment:
Since you are printing this string "==============================\n" again and again, you should store it as a variable or create a function to print this, and call that function instead. It recommended to not use any piece of code repeatedly. Refer to the "DRY" principle.